app/jobs/web_scraping_job.rb

```diff
@@ -44,21 +46,39 @@ def parse_search_results(keyword, html)
     search_result
   end
 
-  def notify_completed(keyword, search_result)
+  def notify_searching(keyword)
     Turbo::StreamsChannel.broadcast_replace_to(
       [keyword.user, :data_set],
       target: "data-container-#{keyword.id}",
       partial: 'keywords/keyword',
-      locals: { keyword:, search_result: }
+      locals: { keyword:, status: 'searching' }
```

Comment:
Wouldn't it be better to add an attribute to the keyword model to track the status?

Reply:
yes! thanks for the suggestion. will make the changes soon.

Reply:
made the changes here #12